schema/3pse/file/applications/espresso/7.2/pw.x/control.json

```diff
@@ -0,0 +1,179 @@
+        },
+        "pseudo_dir": {
+            "type": "string",
+            "description": "directory containing pseudopotential files. Default: value of the $ESPRESSO_PSEUDO environment variable if set; '$HOME/espresso/pseudo/' otherwise"
+        },
+        "tefield": {
+            "type": "boolean",
+            "default": false,
+            "description": "If .TRUE. a saw-like potential simulating an electric field is added to the bare ionic potential. See variables edir, eamp, emaxpos, eopreg for the form and size of the added potential."
+        },
+        "dipfield": {
+            "type": "boolean",
+            "default": false,
+            "description": "If .TRUE. and tefield==.TRUE. a dipole correction is also added to the bare ionic potential - implements the recipe of L. Bengtsson, PRB 59, 12301 (1999). See variables edir, emaxpos, eopreg for the form of the correction. Must be used ONLY in a slab geometry, for surface calculations, with the discontinuity FALLING IN THE EMPTY SPACE."
+        },
+        "lelfield": {
+            "type": "boolean",
+            "default": false,
+            "description": "If .TRUE. a homogeneous finite electric field described through the modern theory of the polarization is applied. This is different from tefield == .true. !"
+        },
+        "nberrycyc": {
+            "type": "integer",
+            "default": 1,
+            "description": "In the case of a finite electric field  ( lelfield == .TRUE. ) it defines the number of iterations for converging the wavefunctions in the electric field Hamiltonian, for each external iteration on the charge density"
+        },
+        "lorbm": {
+            "type": "boolean",
+            "default": false,
+            "description": "If .TRUE. perform orbital magnetization calculation."
+        },
+        "lberry": {
+            "type": "boolean",
+            "default": false,
+            "description": "If .TRUE. perform a Berry phase calculation. See the header of PW/src/bp_c_phase.f90 for documentation"
+        },
+        "gdir": {
+            "type": "number",
+            "description": "For Berry phase calculation: direction of the k-point strings in reciprocal space. Allowed values: 1, 2, 3 1=first, 2=second, 3=third reciprocal lattice vector For calculations with finite electric fields (lelfield==.true.) \"gdir\" is the direction of the field."
+        },
+        "nppstr": {
+            "type": "number",
+            "description": "For Berry phase calculation: number of k-points to be calculated along each symmetry-reduced string. The same for calculation with finite electric fields (lelfield==.true.)."
+        },
+        "gate": {
+            "type": "boolean",
+            "default": false
```

Comment:
There is description for both this and `trism` fields in https://www.quantum-espresso.org/Doc/INPUT_PW.html.